index.html

```diff
@@ -21,11 +23,12 @@
           </section>
      </nav>
   
-
   <div class="container">
```

Comment:
- [ ] Kindly make sure the container of the "Headline" section is a `section` tag. It represents a generic standalone section of the document, so use `section` tag. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.

- [ ] Kindly make sure the content is aligned with `flexbox`. This gives us a lot of options to align all our elements, plus, it's responsive.